src/styles/card.css

```diff
@@ -0,0 +1,60 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 10px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  padding: 0 3px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__header {
+  margin-top: 40px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__stars {
+  display: flex;
+  margin-top: 17px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+  margin-top: 22px;
+}
+
+.price-text {
+  color: #616070;
+  margin: 0;
+}
+
+.price-dolars {
+  font-size: 16px;
+  margin: 0;
+  font-weight: 700;
+}
```

Comment:
these seem like block, in which case mo margins. but i dont think they need to blocks